app/models/ems_refresh/refreshers/ems_refresher_mixin.rb

```diff
@@ -135,13 +135,13 @@ def parse_targeted_inventory(ems, target, collector)
           _log.debug "#{log_header} Parsing inventory..."
           inventory_collections, = Benchmark.realtime_block(:parse_inventory) do
             provider_module = ManageIQ::Providers::Inflector.provider_module(ems.class).name
-            inventory_target_class = "#{provider_module}::Inventory::Target::#{target.class.name.demodulize}".safe_constantize
-            inventory_target = inventory_target_class.new(target)
+            persister_class = "#{provider_module}::Inventory::Persister::#{target.class.name.demodulize}".safe_constantize
```

Comment:
Good point, and I'm chiming in, because the original code was mine, so I'm to blame 😉 
So, if the class in not found, it would return `nil` and `nil.new` will blow up.
So we should assume the class is there, check if its not and raise an exception like `you should really implement this` - so the dev knows what he's missing